Fix inconsistent logging when `pdm use` a different python interpreter
